share/completions/mkvextract.fish

```diff
@@ -0,0 +1,47 @@
+# Sample output of 'mkvmerge -i file.mkv'
+# 
+# File 'file.mkv': container: Matroska
+# Track ID 0: video (MPEG-4p10/AVC/h.264)
+# Track ID 1: audio (AAC)
+# Track ID 2: subtitles (SubStationAlpha)
+# Attachment ID 1: type 'application/x-truetype-font', size 53532 bytes, file name 'some_font.ttf'
+# Chapters: 7 entires
+
+function __fish_print_mkvextract_attachments
+    mkvmerge -i $argv | string match 'Attachment ID*' | string replace -r '.*?(\d+).*? type \'(.*?)\'.*?file name \'(.*?)\'' '$1:\t$3 ($2)'
+end
+
+function __fish_print_mkvextract_tracks
+    mkvmerge -i $argv | string match 'Track ID*' | string replace -r '.*?(\d+): (.*)' '$1:\t$2'
+end
+
+# simple options
+complete -f -c mkvextract -n "test (count (commandline -opc)) -lt 2"    -s V -l "version" -d "Show version information"
+complete -f -c mkvextract -n "test (count (commandline -opc)) -lt 2"    -s h -l "help"    -d "Show help"
+# commands
+complete -f -c mkvextract -n "test (count (commandline -opc)) -lt 2" -a "tracks"       -d "Extract tracks to external files"
```

Comment:
Most of the time, commands support options also before subcommands, i.e. something like `mkvextract --parse-fully tags` would also work. If that is the case here, just counting the number of tokens would break.

Can you confirm if it is?